core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedDocServicePlugin.java

```diff
@@ -207,10 +207,19 @@ private static String normalizeParameterized(Route route) {
 
         final StringBuilder sb = new StringBuilder();
         for (String paramName : route.paramNames()) {
-            final int colonIndex = path.indexOf(':', beginIndex);
-            assert colonIndex != -1;
+            int colonIndex = path.indexOf(':', beginIndex);
+            boolean restPathsPattern = false;
+            if (colonIndex == -1) {
+                colonIndex = path.indexOf('*', beginIndex);
+                assert colonIndex != -1;
+                restPathsPattern = true;
+            }
             sb.append(path, beginIndex, colonIndex);
             sb.append('{');
+            if (restPathsPattern) {
+                // Set a parameterized rest paths pattern "{*foo}".
+                sb.append('*');
+            }
```

Comment:
By the way, could we just use `Route.patternString()` instead of re-assembling the path of `Route`? `/foo/bar/:baz/:*qux` seems readable enough.
What do you think @trustin? 🤔

Reply:
+1

Reply:
+1

Reply:
So.... you mean. can I change the logic of this method to just returning `Route.patternString` instead of re-assembling in this PR ? 🤔  @ikhoon

Reply:
You can just remove `normalizeParameterized()` and replace it with `patternString()`. 
```diff
- builder = EndpointInfo.builder(hostnamePattern, normalizeParameterized(route));
+ builder = EndpointInfo.builder(hostnamePattern, route.patternString());
```